### PR TITLE
Fix/ demo app colors

### PR DESCRIPTION
**Pull request by lidord-wix**
## Description
Fix demo app colors for dark mode

## Changelog
Fix demo app colors for dark mode

### CODE REVIEW

File: demo/src/demoApp.js [hunk 1]
```diff
@@ -42,39 +42,40 @@ function getDefaultNavigationStyle() {
...
       style: 'light',
-      backgroundColor: Colors.primary // for Android
+      backgroundColor: Colors.$backgroundPrimaryHeavy // for Android
     },
     layout: {
-      backgroundColor: Colors.white,
-      orientation: ['portrait', 'landscape']
+      backgroundColor: Colors.$backgroundDefault,
+      orientation: ['portrait', 'landscape'],
+      componentBackgroundColor: Colors.$backgroundDefault
```
Comment (M-i-k-e-l) [inline demo/src/demoApp.js:50]: What's this for?

Comment (lidord-wix) [reply demo/src/demoApp.js:50]: run the demo app and toggle to dark mode (without changing the config file)

Comment (M-i-k-e-l) [reply demo/src/demoApp.js:50]: So you prevent the color change?

**Review by M-i-k-e-l — APPROVED** [review]

File: demo/src/demoApp.js (hunk 1 above)
Comment (lidord-wix) [reply demo/src/demoApp.js:50]: I set the right colors instead the defaults we have today

**Pull request merged**